generators/app/templates/src/app/components/FormInput/index.tsx

```diff
@@ -33,9 +37,13 @@ function FormInput({
   onChange,
   onFocus,
   placeholder = '',
-  readOnly = false
+  readOnly = false,
+  touched,
+  submitCount
 }: Props) {
   const InputComponent = isTextarea ? 'textarea' : 'input';
+  const showError =
+    (touched === undefined || touched) && error && (submitCount === undefined || submitCount > 0);
```

Comment:
Couldn't we assign a default value to `touched` and `submitCount` so we can avoid checking for `undefined`?

Reply:
I think that if you put a value on it, it is not generic.
The idea is to use this behavior regardless of the use of a library for forms.